stack_versions.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+func stackVersions(stackID string) ([]byte, error) {
```

Comment:
this name is misleading, there is no version**S**, this function will wrap stack id string in a json format and returns that content in a byte array. I would name this function as: `stackVersionInfo`

Reply:
or to keep it more generic: archiveInfo -> which then will be stored in an archive_info.json file 👍